src/Search/WishListProductSearchProvider.php

```diff
@@ -174,6 +174,20 @@ private function getProductsOrCount(
                 return [];
             }
 
+            $products = array_map(
+                function ($product) {
+                    if (0 < (int) $product['id_product_attribute']) {
+                        $combinationImages = Product::_getAttributeImageAssociations($product['id_product_attribute']);
+                        if (count($combinationImages)) {
```

Comment:
```suggestion
                        if (isset($combinationImages[0])) {
```
Always test what you use. Never trust an array.